docs/source/developing/writing-docs.adoc

```diff
@@ -110,4 +183,7 @@ $ cd ..
 $ rake build
 ----
 
-If the build completes successfully, the site is available under _preview/openshift-origin/latest/welcome/index.html_.
+If the build completes successfully, the site is available under *_preview/openshift-origin/latest/welcome/index.html_*.
+
+OpenShift runs a nightly build in which they update the documentation. After you update the documentation,
```

Comment:
That's not correct. This has nothing to do with local docs development. This is a production/release thing.